packages/react-components/src/components/CaptionsSettingsModal.tsx

```diff
@@ -347,7 +347,9 @@ export const CaptionsSettingsModal = (props: CaptionsSettingsModalProps): JSX.El
           styles={CaptionsSettingsModalStyle}
         >
           <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={titleContainerClassName}>
-            <Text className={titleClassName}>{strings?.captionsSettingsModalTitle}</Text>
+            <Text className={titleClassName} role="heading">
```

Comment:
I think it needs aria-level too

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/heading_role

![image](https://github.com/user-attachments/assets/91e5c50e-cc0d-4709-a046-a040f77eab63)